src/Test.UnitTests.Sarif.Multitool.Library/Test.UnitTests.Sarif.Multitool.Library.csproj

```diff
@@ -13,28 +13,7 @@
     <IsTestProject>true</IsTestProject>
     <RootNamespace>Microsoft.CodeAnalysis.Sarif.Multitool</RootNamespace>
   </PropertyGroup>
-
-  <ItemGroup>
-    <None Remove="TestData\ConvertCommand\SemmleQlSample.csv" />
-    <None Remove="TestData\ExportRuleDocumentationCommand\ExpectedOutputs\MarkdownFullDescription.md" />
-    <None Remove="TestData\ExportRuleDocumentationCommand\ExpectedOutputs\MarkdownShortDescription.md" />
-    <None Remove="TestData\ExportRuleDocumentationCommand\ExpectedOutputs\NoDescription.md" />
-    <None Remove="TestData\ExportRuleDocumentationCommand\ExpectedOutputs\NonStandardMessageStringKey.md" />
-    <None Remove="TestData\ExportRuleDocumentationCommand\ExpectedOutputs\StandardMessageStringKey.md" />
-    <None Remove="TestData\ExportRuleDocumentationCommand\ExpectedOutputs\TextShortDescription.md" />
-    <None Remove="TestData\MergeCommand\ExpectedOutputs\DuplicatedResults.sarif" />
-    <None Remove="TestData\MergeCommand\ExpectedOutputs\NoInputFiles.sarif" />
-    <None Remove="TestData\MergeCommand\Inputs\DuplicatedResults.sarif" />
-    <None Remove="TestData\PageCommand\elfie-arriba - Copy.sarif" />
-    <None Remove="TestData\PageCommand\elfie-arriba.sarif" />
-    <None Remove="TestData\RebaseUriCommand\ExpectedOutputs\RunWithArtifacts.sarif" />
-    <None Remove="TestData\RebaseUriCommand\RunWithArtifacts.sarif" />
-    <None Remove="TestData\ValidateCommand\Configuration.json" />
-    <None Remove="TestData\ValidateCommand\Configuration.xml" />
-    <None Remove="TestData\ValidateCommand\ValidateSarifOneZeroZero.sarif" />
-    <None Remove="xunit.runner.json" />
-  </ItemGroup>
```

Comment:
></ItemGroup> [](http://example.com/codeflow?start=2&length=12)

Michael: This is junk that VS insists on adding. You can see it's not in sync w/what's below, let's just delete it.

Reply:
removed